projectile.el

```diff
@@ -1489,8 +1497,22 @@ Fallback to a generic command when not in a VCS-controlled project."
     ('svn projectile-svn-command)
     ('sapling projectile-sapling-command)
     ('jj projectile-jj-command)
+    ('osc projectile-osc-command)
     (_ projectile-generic-command)))
 
+
+(defun projectile-osc-command-files (&optional root)
+  "Return files of osc vcs, return either packages or files belonging to package."
+  (or root (setq root default-directory))
+  (let* ((files_ (car (xml-parse-file (if (file-exists-p ".osc/_files")
```

Comment:
Why `files_` instead of `files`?

Reply:
It seems to me a `cond` might be a better fit than the nested `if`s you've opted for.

Reply:
> Why `files_` instead of `files`?

Because the file containing the content is called `_files` and `_files` as variable name didn't exactly work.